react/client/src/components/Textarea.tsx

```diff
@@ -28,27 +37,75 @@ interface TextFieldProps {
   handleChange: (e: React.ChangeEvent<HTMLInputElement>) => void;
   inputName: string;
   placeholder: string;
+  multi: boolean;
 }
 
 const MultilineTextFields: React.FC<TextFieldProps> = ({
   handleChange,
   inputName,
   placeholder,
+  multi,
 }) => {
+  const checkValid = (e: string) => {
+    isValid(e.length > 0);
+  };
+  const [valid, isValid] = useState(false);
   const classes = useStyles();
-  return (
+  return inputName === 'age' ? (
```

Comment:
Haha `Input` and `Textarea` are two different things - I would like to see them as separate components. 

On the web the function very differently with one being a single line and the other be multiline. There's also implications for the html itself, such as there being the `value` attribute on an input element.

Reply:
Okay! Updating now.